pkg/node/node.go

```diff
@@ -921,7 +921,8 @@ func NewBee(interrupt chan struct{}, sysInterrupt chan os.Signal, addr string, p
 
 	pinningService := pinning.NewService(storer, stateStore, traversalService)
 
-	pushSyncProtocol := pushsync.New(swarmAddress, nonce, p2ps, storer, kad, tagService, o.FullNodeMode, pssService.TryUnwrap, validStamp, logger, acc, pricer, signer, tracer, warmupTime)
```

Comment:
I wonder if we can get away with not even registering the protocol for bootnodes 🤔

Reply:
@aloknerurkar @mrekucci @notanatol wdyt?

Reply:
> @aloknerurkar @mrekucci @notanatol wdyt?

we could do that but I feel that having a switch in place is a more "fine grained" solution

Reply:
Sorry for not getting back!

So @istae is right. This will only disable storing on pushsync. However we will still see pushsync protocol traffic. I think the intention of this issue is to not have that as well. I think it would be better to disable the protocol itself, similar to how we do for pullsync i.e. not register the protocol on the libp2p host.

@janos Could there be any issues if we dont register this protocol? We might start seeing more errors for pushsync? Or would the nodes understand that this peer doesnt have the protocol and not send messages?

Reply:
@vladopajic let's go with p2p.WithBlocklistStreams